libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergEditorFragment.java

```diff
@@ -333,6 +343,30 @@ public void onQueryCurrentProgressForUploadingMedia() {
                         updateMediaProgress();
                     }
                 },
+                new OnSetFeaturedImageListener() {
+                    @Override
+                    public void onSetFeaturedImageButtonClicked(int mediaId) {
+                        if (mediaId == mFeaturedImageId) {
+                            // nothing special to do, trying to set the image that's already set as featured
+                            return;
+                        }
+
+                        if (mediaId == 0) {
```

Comment:
I am thinking we could hardcode this `0` which serves as the `DEFAULT_MEDIA_ID` so it's easier to identify its purpose and if we need to change its value it is in a reusable state.

Reply:
Good point about naming the special value. In this case, I guess it's probably closer to a `MEDIA_ID_NO_FEATURED_IMAGE_SET` that denotes the special interpretation of the value, right?

Reply:
That makes a lot of sense! I've introduced that constant in https://github.com/wordpress-mobile/WordPress-Android/pull/14503/commits/9630c2955da873dbbd033849a6d1494876bbb582. I've gone ahead to use the `MEDIA_ID_NO_FEATURED_IMAGE_SET` suggestion for naming and used it in three different places, where there were checks for zero. How does that seem?

Reply:
> That makes a lot of sense! I've introduced that constant in 9630c29. I've gone ahead to use the MEDIA_ID_NO_FEATURED_IMAGE_SET suggestion for naming and used it in three different places, where there were checks for zero. How does that seem?

Looking good @SiobhyB

Reply:
Nice! Any opportunities to introduce the const (or define in a common place and reuse) on the "sending" side as well? Like, in the places where functions _use_ the `0` to denote clearing the setting, no just when checking against it.

Reply:
I've gone ahead to make some changes to the button on the JS side here https://github.com/WordPress/gutenberg/pull/31705#pullrequestreview-667361512

If there's a better way for defining the constant there, I'm all ears. :D

Reply:
Looks like all comments/ideasof this thread have been addressed so, I'll mark it as resolved :)